helm/answer-coalesce/values.yaml

```diff
@@ -46,6 +46,8 @@ redis:
     size:
     class:
 
+dataUrl: "https://stars.renci.org/var/answer_coalesce/2021-09-24.rdb"
```

Comment:
there is actually a better version of this redis at /projects/stars/Data_services/AC/2021-09-25.rdb

the permissions on the /var directory didnt permit me to copy it there.

Reply:
ok will copy those in and update the url

Reply:
thanks!